in_toto/verifylib.py

```diff
@@ -1113,13 +1199,16 @@ def in_toto_verify(layout_path, layout_key_paths):
   log.info("Verifying link metadata signatures...")
   verify_all_steps_signatures(layout, chain_link_dict)
 
+  log.info("Verifying sublayouts...")
+  chain_link_dict = verify_sublayouts(layout, chain_link_dict)
```

Comment:
Please add this to the docstring actions above. Please also explain why sublayouts are verified at this place (after verify signatures and before the rest), you can explain this either in the docstring or as comment.